<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainClient</class>
 <widget class="QMainWindow" name="MainClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon.png</normaloff>../icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-image: url(C:/Users/ouniy/Desktop/projet c++/bg.jpg);
    background-repeat: no-repeat;
    background-position: center;
	
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>701</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget {
    background-color: #f2f2f2; /* Background color for the tab widget */
}

QTabBar::tab {
    background-color: #4CAF50; /* Green background color for each tab */
    color: white; /* Text color for each tab */
    padding: 8px; /* Adjust padding as needed */
}

QTabBar::tab:selected {
    background-color: #45a049; /* Darker green background color for the selected tab */
}

QTabWidget::pane {
    border: 10px solid #4CAF50; /* Green border around the tab widget */
}
QLineEdit {
    padding: 2px;
    border: 2px solid #4CAF50; /* Green border */
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid #45a049; /* Darker green border on focus */
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px; /* Adjust the font size as needed */
    font-weight: bold;
    color: #FFFFFF; /* Orange text color */
}
</string>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px; /* Adjust the font size as needed */
    font-weight: bold;
    color: #FFFFFF; /* Orange text color */
}
</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px; /* Adjust the font size as needed */
    font-weight: bold;
    color: #FFFFFF; /* Orange text color */
}
</string>
      </property>
      <property name="text">
       <string>Pr√©nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    padding: 2px;
    border: 2px solid #4CAF50; /* Green border */
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid #45a049; /* Darker green border on focus */
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    padding: 2px;
    border: 2px solid #4CAF50; /* Green border */
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid #45a049; /* Darker green border on focus */
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    padding: 2px;
    border: 2px solid #4CAF50; /* Green border */
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid #45a049; /* Darker green border on focus */
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    font-size: 16px;
    margin: 4px 2px;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    padding: 2px;
    border: 2px solid #4CAF50; /* Green border */
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid #45a049; /* Darker green border on focus */
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px; /* Adjust the font size as needed */
    font-weight: bold;
    color: #FFFFFF; /* Orange text color */
}
</string>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px; /* Adjust the font size as needed */
    font-weight: bold;
    color: #FFFFFF; /* Orange text color */
}
</string>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px; /* Adjust the font size as needed */
    font-weight: bold;
    color: #FFFFFF; /* Orange text color */
}
</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px; /* Adjust the font size as needed */
    font-weight: bold;
    color: #FFFFFF; /* Orange text color */
}
</string>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px; /* Adjust the font size as needed */
    font-weight: bold;
    color: #FFFFFF; /* Orange text color */
}
</string>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    font-size: 16px;
    margin: 4px 2px;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>611</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    background-color: white; /* Background color for the table view */
    border: 2px solid #4CAF50; /* Green border around the table view */
    selection-background-color: #45a049; /* Highlight color on item selection */
	border-radius: 5px;
}

QTableView::item {
    padding: 5px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    padding: 8px;
    border: 2px solid #4CAF50; /* Green border */
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid #45a049; /* Darker green border on focus */
}
</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    border: 2px solid #4CAF50; /* Green border */
    border-radius: 5px;
    padding: 5px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: center right;
    width: 20px; /* Set the width of the drop-down button */
    border-left: 2px solid #4CAF50; /* Match the QComboBox border */
}

QComboBox QAbstractItemView {
    border: 2px solid #4CAF50; /* Match the QComboBox border */
    background-color: white; /* Background color for the drop-down list */
    selection-background-color: #45a049; /* Highlight color on item selection */
}
</string>
      </property>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prenom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Age</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>7</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    font-size: 16px;
    margin: 4px 2px;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>290</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    padding: 8px;
    border: 2px solid #4CAF50; /* Green border */
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid #45a049; /* Darker green border on focus */
}
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
      </property>
      <property name="text">
       <string>Camera</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px; /* Adjust the font size as needed */
    font-weight: bold;
    color: #FFFFFF; /* Orange text color */
}
</string>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>137</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    font-size: 16px;
    margin: 4px 2px;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
    </property>
    <property name="text">
     <string>Accueil</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
    </property>
    <property name="text">
     <string>Factures</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
    </property>
    <property name="text">
     <string>Reservations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
    </property>
    <property name="text">
     <string>Employe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: 	#429156; /* Green */
    color: white;
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;
    font-weight: bold; /* Make the text bold */
}

QPushButton:hover {
    background-color: #45a040; /* Darker green on hover */
}
</string>
    </property>
    <property name="text">
     <string>Voitures</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
